lib/adapters/PgsqlAdapter.php

```diff
@@ -75,7 +75,7 @@ public function create_column(&$column)
 		$c->inflected_name	= Inflector::instance()->variablize($column['field']);
 		$c->name			= $column['field'];
 		$c->nullable		= ($column['not_nullable'] ? false : true);
-		$c->pk				= ($column['pk'] ? true : false);
+		$c->pk				= ($column['pk'] == 't');
```

Comment:
See https://github.com/jpfuentes2/php-activerecord/blob/master/lib/adapters/PgsqlAdapter.php#L45-L50 here select value t, and default bool value in postrges is 't' or 'f' http://www.postgresql.org/docs/9.4/static/datatype-boolean.html#DATATYPE-BOOLEAN-EXAMPLE